sklearn/dummy.py

```diff
@@ -63,19 +64,19 @@ class DummyClassifier(MultiOutputMixin, ClassifierMixin, BaseEstimator):
 
     Attributes
     ----------
-    classes_ : array or list of array of shape (n_classes,)
+    classes_ : ndarray or list of ndarray of shape (n_classes,)
```

Comment:
ndarray or list -> array like

(Though did you make sure it can actually be a list?)

Reply:
I have changed `classes_`, `n_classes_` and `class_prior_` to distinguish between single output problem and multi-output problem.